sync/commit.py

```diff
@@ -483,7 +495,11 @@ def _apply_patch(patch,  # type: bytes
                     logger.info(e)
             try:
                 logger.info("Creating commit")
-                return Commit.create(dest_repo, msg, None, amend=amend, author=author)
+                commit_kwargs = {}
+                if allow_empty:
+                    commit_kwargs["allow_empty"] = True
+                return Commit.create(dest_repo, msg, None, amend=amend, author=author,
+                                     **commit_kwargs)
```

Comment:
if I didn't miss anything, this block of code will return:
`{}` or `{"allow_empty": True}`

```
>>> def create(blah, allow_empty=False):
...     print(blah, allow_empty)
... 
>>> commit_kwargs = {}
>>> create('ok', **commit_kwargs)
ok False
>>> commit_kwargs = {"allow_empty": True}
>>> create('ok', **commit_kwargs)
ok True
```

But the dictionary has only one key:value, always.  AND `allow_empty` is not redefined in `_apply_patch` 
https://github.com/mozilla/wpt-sync/pull/740/files#diff-fe9d52c31c7eead9d4c80279ffa4af61R393-R407

so `allow_empty` will be already `False` if not changed in the call to `_apply_path` and `True` if explicitly given in the function.

So this seems it would just work the same.

```suggestion
                return Commit.create(dest_repo, msg, None, amend=amend, author=author, allow_empty)
```

Reply:
Yep, not sure why I didn't do that in the first place (can't use a positional argument after kwargs ofc).